graspologic/embed/n2v.py

```diff
@@ -22,6 +22,7 @@ def node2vec_embed(
     workers: int = 8,
     iterations: int = 1,
     interpolate_walk_lengths_by_node_degree: bool = True,
+    random_seed: Optional[int] = None,
```

Comment:
I think we are missing a docstring for this parameter in this method.

Reply:
I will add it in the next commit.
But regarding your previous comment @Nyecarr , we do need the embedding deterministic between different processes right?

Reply:
It would be nice for sure. That being said I don't think we support it in our spectral embedding methods so I personally wouldn't hold this pull request up over it. However, if you feel inclined that would be a great contribution!

Reply:
Ok Thanks. I also tried with `RandomState` but I am still struggling to keep the output deterministic between different processes.
I will go through `kmeans` implementation once. If you @Nyecarr think that there is something I should look into, please let me know.

Reply:
This inconsistency for deterministic output between different processes is probably due to `Word2Vec` in function `_learn_embeddings` which also suggests using `workers=1` and `PYTHONHASHSEED` for reproducibility.
I will try to see if I can find a solution to this.

Reply:
@Nyecarr This can be implemented. But the only thing is the user has to set the `PYTHONHASHSEED` environment variable before starting the python environment.
Should we implement it? Maybe we can provide a warning to the user if the env variable is not set, saying that if its not set then the results won't be deterministic between different processes.

Reply:
Wow, this seems crazy to me. To be clear, I'm not saying you're saying anything wrong, I just wouldn't have expected this!  Let me think about this a bit and get back to you, but thank you for spending so much time on this - it's a great help!

Reply:
I was also surprised, I thought just adding a seed would do the trick. Anyway, I would suggest going through [Word2Vec](https://radimrehurek.com/gensim/models/word2vec.html). Or function `_learn_embeddings` in the `n2v.py` where we are using Word2Vec for the model.
Please let me know if you find something otherwise :)